raiden/tests/unit/test_tokennetwork.py

```diff
@@ -188,11 +189,131 @@ def test_multiple_channel_states(
     )
 
     mediated_transfer = make_receive_transfer_mediated(
+        channel_state=channel_state,
+        privkey=pkey,
+        nonce=1,
+        transferred_amount=0,
+        lock=lock,
+        token_network_address=token_network_state.address,
+    )
+
+    from_route = factories.route_from_channel(channel_state)
+    init_target = ActionInitTarget(
+        from_route,
+        mediated_transfer,
+    )
+
+    node.state_transition(chain_state, init_target)
+
+    closed_block_number = open_block_number + 10
+    channel_close_state_change = ContractReceiveChannelClosed(
+        channel_state.partner_state.address,
+        token_network_state.address,
+        channel_state.identifier,
+        closed_block_number,
+    )
+
+    channel_closed_iteration = token_network.state_transition(
+        payment_network_identifier,
+        channel_new_iteration.new_state,
+        channel_close_state_change,
+        pseudo_random_generator,
+        closed_block_number,
+    )
+
+    settle_block_number = closed_block_number + channel_state.settle_timeout + 1
+    channel_settled_state_change = ContractReceiveChannelSettled(
+        our_address,
+        token_network_state.address,
+        channel_state.identifier,
+        settle_block_number,
+    )
+
+    channel_settled_iteration = token_network.state_transition(
+        payment_network_identifier,
+        channel_closed_iteration.new_state,
+        channel_settled_state_change,
+        pseudo_random_generator,
+        closed_block_number,
+    )
+
+    token_network_state_after_settle = channel_settled_iteration.new_state
+    ids_to_channels = token_network_state_after_settle.channelidentifiers_to_channels
+    assert len(ids_to_channels) == 1
+    assert channel_state.identifier in ids_to_channels
+
+    unlock_blocknumber = settle_block_number + 5
+    channel_batch_unlock_state_change = ContractReceiveChannelBatchUnlock(
+        transaction_from=address,
+        token_network_identifier=token_network_state.address,
+        participant=our_address,
+        partner=address,
+        locksroot=lock_secrethash,
```

Comment:
Not so sure about this, please check.

Reply:
This looks correct, there was only one transfer, therefor the merkle tree has only one element, and it's root hash will be equal to the hash of that single element.